php/Terminus/Commands/SiteCommand.php

```diff
@@ -918,48 +918,63 @@ public function hostnames($args, $assoc_args) {
    * ## OPTIONS
    *
    * [--site=<site>]
-   * : Site to use
+   * : Site to import content to
+   *
+   * [--env=<env>]
+   * : Name of the environment to import to
    *
    * [--url=<url>]
    * : URL of archive to import
    *
    * [--element=<element>]
-   * : Site element to import (i.e. code or database)
+   * : Site element to import. Options are files or database.
    *
    * @subcommand import-content
    */
-  public function import($args, $assoc_args) {
-    $site = $this->sites->get($this->input()->siteName(array('args' => $assoc_args)));
-    $url  = $this->input()->string(
-      array(
+  public function importContent($args, $assoc_args) {
+    $site = $this->sites->get(
+      $this->input()->siteName(['args' => $assoc_args,])
+    );
+    if (isset($assoc_args['env'])) {
+      $env_name = $assoc_args['env'];
+    } else {
+      $env_name = $this->input()->env(
+        [
+          'args'    => $assoc_args,
+          'choices' => array_diff($site->environments->ids(), ['test', 'live',]),
+        ]
+      );
+    }
+    $env = $site->environments->get($env_name);
+    $url = $this->input()->string(
+      [
         'args'    => $assoc_args,
         'key'     => 'url',
-        'message' => 'URL of archive to import'
-      )
+        'message' => 'URL of archive to import',
+      ]
     );
     if (!$url) {
       $this->log()->error('Please enter a URL.');
     }
 
     if (!isset($assoc_args['element'])) {
-      $element_options = array('database', 'files');
-      $element_key     = $this->input()->menu(
-        array(
-          'choices' => $element_options,
+      $element_key = $this->input()->menu(
+        [
+          'choices' => ['database', 'files',],
```

Comment:
In the `backups` command we use `db` instead of the longer `database`: https://github.com/pantheon-systems/terminus/blob/79159a10f07f40cc32bb59f51c869d0d1c2a148d/php/Terminus/Commands/SiteCommand.php#L50